spec/bump/cli_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+
+describe Bump::CLI do
+  around(:each) do |example|
+    modified_path = File.expand_path("../../..", __FILE__) + "/exe:" + ENV["PATH"]
+    ClimateControl.modify PATH: modified_path do
+      example.run
+    end
+  end
+
+  describe "validate command" do
+    it "outputs the right thing" do
+      expect do
+        print `bump validate`
```

Comment:
Is the `print` necessary? I don't think so as the matcher captures the stream https://github.com/rspec/rspec-expectations/blob/master/lib/rspec/matchers/built_in/output.rb

Reply:
Yep, it is: `expected block to output "Deploying...\n" to stdout, but output nothing` without it.